advanced_functionality/scikit_bring_your_own/scikit_bring_your_own.ipynb

```diff
@@ -493,6 +499,88 @@
    "source": [
     "sess.delete_endpoint(predictor.endpoint)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Batch Transform Job\n",
+    "One can use trained model to get inference on large data sets by using [Amazon SageMaker Batch Transform](https://docs.aws.amazon.com/sagemaker/latest/dg/how-it-works-batch.html). A Batch Transform Job takes your input data S3 location and publishes prediction result to specified S3 output folder. Similar to hosting, lets extract inferences for training data to demo Batch Transform mechanism."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a Transform Job\n",
+    "We'll create an `Transformer` that defines how to use the container to get inference results on a data set. This includes the configuration we need to invoke SageMaker Batch transform:\n",
+    "\n",
+    "* The __instance count__ which is the number of machines to use to extract inferences\n",
+    "* The __instance type__ which is the type of machine to use to extract inferences\n",
+    "* The __output path__ determines where the inference results will be written"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transform_output_folder = \"batch-transform-output\"\n",
+    "output_path=\"s3://{}/{}\".format(sess.default_bucket(), transform_output_folder)\n",
+    "\n",
+    "transformer = tree.transformer(instance_count=1,\n",
+    "                               instance_type='ml.m4.xlarge',\n",
+    "                               output_path=output_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use tranform() on the transfomer to get inference results against the data that we uploaded above. We provide below options when invoking transformer. \n",
```

Comment:
"We use tranform() on the transfomer to get inference results against the data that we uploaded. You can use these options when invoking the transformer. \n",